app/src/main/res/values-sw600dp-land/booleans.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="faq_single_activity_scroll_fill_view_port">true</bool>
+</resources>
```

Comment:
Add EOF

Reply:
@prayutsu I have added EOF please review it.